ui/jobs/jobs.css

```diff
@@ -1,3 +1,7 @@
+:root {
+  --proc-box-flash-repeat: infinite;
```

Comment:
I'm wondering if there is any benefit defining this here instead of just providing a default in case the variable isn't defined like done for the other parameters. As this isn't reused anywhere else, I don't really see any benefit. Any opinions?

Reply:
I don't have the ability to test this right now. Does this affect in any way the ability to override these values? If not, then I'm inclined to say that they can all use the default syntax like you have in other areas

Reply:
The way I read the documentation, no. Default is simply used if there is no definition for the variable.